clients/go/cmd/zbctl/internal/commands/updateJobTimeout.go

```diff
@@ -0,0 +1,68 @@
+// Copyright © 2018 Camunda Services GmbH (info@camunda.com)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package commands
+
```

Comment:
@ana-vinogradova-camunda the linter believes this and updateRetries are the similar. Which is the case if we replace `retries` with `timeout`. Which is a good sign in a way.

I would disable the linter on this file with the comment
//nolint:dulp after the line 15

Reply:
Still failing, not sure what am I doing wrong.

Reply:
the line should simply 

```
//nolint:dupl
```